specification/compute/resource-manager/readme.md

```diff
@@ -168,6 +168,20 @@ directive:
       - DefinitionsPropertiesNamesCamelCase
 
 ```
+### Tag: package-2018-10-01
+
+These settings apply only when `--tag=package-2018-10-01` is specified on the command line.
+
+``` yaml $(tag) == 'package-2018-10-01'
+input-file:
+- Microsoft.Compute/stable/2018-10-01/compute.json
+- Microsoft.Compute/stable/2018-06-01/runCommands.json

```

Comment:
Update runcommand to 2018-10-01 version.  RunCommand should always be the same as Compute

Reply:
Fixed